test/spec/protractor_sync_test.ts

```diff
@@ -555,11 +555,13 @@ describe('Protractor extensions', () => {
     }));
 
     it('times out', createTest(() => {
-      let counter = 0;
+      let expectationMethodCheckCount = 0;
 
       expect(() =>
-        protractorSync.polledExpect(() => counter++, 100).toBeLessThan(0)
+        protractorSync.polledExpect(() => expectationMethodCheckCount++, 100).toBeLessThan(0)
       ).toThrowError(/Expected \d+ to be less than 0\./);
+      expect(expectationMethodCheckCount).toBeGreaterThan(9);
```

Comment:
Minor:  When I ran the this test 1000 times,  I received (albeit only twice) the error:
``` Expected 9 to be greater than 9.``` 

 Changing this to 8 I never saw the error in 1000 times.